specification/cognitiveservices/data-plane/Common/Parameters.json

```diff
@@ -1,39 +1,61 @@
 {
- "swagger": "2.0",
- "info": {
-  "version": "2017-08-30",
-  "title": "Common Referenced Parameters File",
-  "description": "File containing commonly referenced parameters."
- },
- "paths": {},
- "parameters": {
-  "AzureRegion": {
-   "name": "AzureRegion",
-   "description": "Supported Azure regions for Cognitive Services endpoints",
-   "x-ms-parameter-location": "client",
-   "required": true,
-   "type": "string",
-   "in": "path",
-   "x-ms-skip-url-encoding": true,
-   "x-ms-enum": {
-    "name": "AzureRegions",
-    "modelAsString": false
-   },
-   "enum": [
-    "westus",
-    "westeurope",
-    "southeastasia",
-    "eastus2",
-    "westcentralus"
-   ]
-  },
-  "SubscriptionKey": {
-   "name": "Ocp-Apim-Subscription-Key",
-   "x-ms-client-name": "SubscriptionKey",
-   "in": "header",
-   "required": true,
-   "description": "Subscription key in header",
-   "type": "string"
-  }
- }
+    "swagger": "2.0",
+    "info": {
+        "version": "2017-08-30",
+        "title": "Common Referenced Parameters File",
+        "description": "File containing commonly referenced parameters."
+    },
+    "paths": {},
+    "parameters": {
+        "AzureRegion": {
```

Comment:
@fearthecowboy @DavidLiCIG Shouldn't this be removed from here, since it's in BasicRegion and ExtendedRegion?